mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -660,6 +651,25 @@ func resourceDataprocCluster() *schema.Resource {
 											},
 										},
 									},
+									"confidential_instance_config": {
+										Type:         schema.TypeList,
+										Optional:     true,
+										AtLeastOneOf: gceClusterConfigKeys,
+										Computed:     true,
+										MaxItems:     1,
+										Description:  `Confidential Instance Config for clusters using Confidential VMs.`,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"enable_confidential_compute": {
+													Type:         schema.TypeBool,
+													Optional:     true,
+													AtLeastOneOf: schieldedInstanceConfigKeys,
```

Comment:
Is `AtLeastOneOf: schieldedInstanceConfigKeys,` intentional?

Reply:
Looks like that was a by mistake.